src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    @extend %transition;
```

Comment:
you dont need transform transition here, so you dont need that extend
just move time to variable